### PR TITLE
change_before_release2.1.1

### PR DESCRIPTION
**Pull request by chengzhuzhang**
A micro release to resolve an issue likely stems from `cartopy0.18` https://github.com/E3SM-Project/e3sm_diags/issues/318

### CODE REVIEW

**Comment by chengzhuzhang** [comment]
@xylar It's a draft PR. I hope to also fix the yaml issue in this PR. I will pin you after fixing them.

File: conda/meta.yaml [hunk 1]
```diff
@@ -38,6 +36,7 @@ requirements:
     - cartopy
     - beautifulsoup4
     - lxml
+    - dask =2.15.0
```
Comment (xylar) [inline conda/meta.yaml:39]:
```suggestion
    - dask 2.15.0
```

File: conda/meta.yaml [hunk 2]
```diff
@@ -8,11 +8,9 @@ package:
 source:
     url: https://github.com/E3SM-Project/e3sm_diags/archive/v{{ version }}.tar.gz
     sha256: 9c169c27fcdfec870b4e20fd71c24a3a426136424c32003bb9a714af157a3cb5
-    patches:
-      - 0001-one-more-change-on-version-before-releasing.patch
```
Comment (xylar) [inline conda/meta.yaml:12]: Go ahead and delete the patch file as well.

**Comment by chengzhuzhang** [comment]
@xylar I have addressed the review comments. Let me know if the PR can be merged. I will upate yaml files after releasing on anaconda. Thank you!

**Comment by xylar** [comment]
@chengzhuzhang, do you need me to build to avoid the `python_abi` issue?

**Comment by chengzhuzhang** [comment]
@xylar was just about to ask your help on building... I'm releasing on github now

**Comment by chengzhuzhang** [comment]
It is now released :) 
https://github.com/E3SM-Project/e3sm_diags/releases/tag/v2.1.1

**Comment by xylar** [comment]
Okay, I just uploaded the new build.  We, of course, need to update the hashtag in the recipe.  I'll make a PR for that.

**Pull request merged**